### ISSUE TITLE
Does not update correct editor when there are multiple editors on the page.

### ISSUE THREAD

**Issue by sanelson2000** [melquibrito/source-code-editor-tinymce-plugin#1]
I have a page with 2 tinymce editors on it.  And just installed your code editor plugin (which is very nice, BTW).  If I try to use the code editor plugin on the first editor, when I click Save, it overwrites the contents of the second editor with the value instead of updating the first.  It appears that tinymce.activeEditor is not always returning the editor that the plugin was called from.  I hacked around it by passing the editor variable to displayToxEditorModal and setting the id of the editor as a data attribute on the modal, which I then read back in saveContent()... But I'm sure there's probably a cleaner way to do it.

```javascript
function displayToxEditorModal(e, display = true) {
  let el = document.getElementById('tox-codeeditor-wrap');
  if (display) {
    el.dataset.editor = e.id;
    el.style.display = "flex";
    el.focus();
    document.body.classList.add('tox-codeeditor__disable-scroll');
  } else {
    e = e || tinymce.get(el.dataset.editor);
    el.style.display = "none";
    document.body.classList.remove('tox-codeeditor__disable-scroll');
    e.focus();
  }
}

function saveContent() {
  let el = document.getElementById('tox-codeeditor-wrap');
  let e = tinymce.get(el.dataset.editor);
  e.focus();
  e.undoManager.transact(function() {
    e.setContent(aceEditor.getValue())
  });
  e.selection.setCursorLocation();
  e.nodeChanged();
  displayToxEditorModal(e,false);
}
```

**Comment by prathamVaidya** [comment]
@sanelson2000  I don't think this plugin is maintained any further. For anyone landing here in future, Try  [supercode](https://github.com/prathamVaidya/supercode-tinymce-plugin) plugin that I created for tinyMCE editor. It fixes all these issues and offers more.

![chrome-capture-2024-4-7 (1)](https://github.com/prathamVaidya/supercode-tinymce-plugin/assets/61202986/eb519154-ac86-423a-8d63-450f174c7e4b)
